rpp-charter.md

```diff
@@ -1,45 +1,26 @@
-# Background and Problem Space
+# Background and Motivation
+The Extensible Provisioning Protocol (EPP) was standardized (STD69) over 20 years ago to address the needs of domain name management between domain name registries and registrars. It encodes protocol Request and response using XML and can be carried over multiple transport protocols. Though EPP has served the domain name industry well, there is now a desire to modernize provisioning protocol using a RESTFUL design and JSON encoding. Such design could take advantage of stateless architecture and widely deployed solutions such as OpenAPI with documentation, testing and code generation tools,L4-L7 network services.
```

Comment:
We probably need discussion about allowing the scope to encompass things like OpenAPI, JSON Schema, etc... But this is a good point.

Reply:
Yes, the change proposed here relfects the conclusion of discussion with Pawel Kowalik.

Reply:
I believe this has been covered in another PR.